components/omega/doc/devGuide/Field.md

```diff
@@ -49,8 +50,15 @@ for some intermediate calculations or unique analyses. If there is no
 restriction on valid range, an appropriately large range should be provided for
 the data type. Similarly, if a FillValue is not being used, a very unique
 number should be supplied to prevent accidentally treating valid data as a
-FillValue.  Actual field data stored in an array is attached in a separate
-call as described below. Fields without a data array can be created with:
+FillValue. The optional TimeDependent argument can be omitted and is assumed
+to be true by default. Fields with this attribute will be output with the
+unlimited time dimension added. Time should not be added explicitly in the
+dimension list since it will be added during I/O. Fields that do not change
+with time should include this argument with the value false so that the time
+dimension is not added. Actual field data stored in an array is attached in a
+separate call as described below. Scalar fields can be added by setting the
+NumDims to zero (the DimNames is then ignored). Scalar data is attached using
```

Comment:
I modified the documentation to clarify and fix the variable name. I could make the Dimensions arg a pointer rather than a reference and we could pass a nullptr for scalars if that's preferable. I don't know how often we'll have scalars as fields besides time and perhaps time-series diags.

Reply:
I think the current implementation is better, given we'll use this scalar functionality pretty seldom.